bin/rose-make-docs

```diff
@@ -65,6 +65,30 @@
 #     `linkcheck`
 #         Checks external links.
 #
+# SOFTWARE ENVIRONMENT
+#     The software dependencies can be found by running the command::
+#
+#        $ rose check-software --docs
+#
+#     Rose provides two ways of installing the Python dependencies for the Rose
+#     documentation builder:
+#
+#     Virtual Environment:
+#         Rose will automatically build a Python virtual environment
+#         when the --venv flag is used with this command. The virtual
+#         environment will be created in rose/venv and will be destroyed
+#         after use. Use the --dev flag to prevent the environment being
+#         destroyed for future use. Example::
+#
+#            $ rose make-docs --venv --dev html  # create and keep a virtualenv
+#
+#     Conda
+#         An environment file for creating a conda env can be found in
+#         etc/rose-docs-env.yaml. Example usage::
+#
+#            $ conda env create -f etx/rose-docs-env.yaml  # create conda env
```

Comment:
Whilst just documenting (& as an example), for future conda support I should point out:
1. Typo: should be ``etc/``
2. Perhaps append ``-p $ROSE_HOME`` to this command to ensure the file search path is tethered?

Reply:
1. Yep
2. Dunno, not all users will have `ROSE_HOME` defined in their environment (within rose commands it is ok to use such environment variables as they are provided by `bin/rose`). Because this command is illustrative rather than functional I think a relative path is OK, I'm not sure what the alternative would be?

Reply:
Fair enough!